lib/versions.js

```diff
@@ -43,7 +43,7 @@ function normalizeVersion(release) {
         .value();
 
     return {
-        tag: normalizeTag(release.tag_name),
+        tag: normalizeTag(release.tag_name).split('-')[0],
```

Comment:
@SamyPesse I've figured it out again. Without this change, each "release" object (with keys `tag`, `channel`, `notes`, `published_at` ... (when you look at /api/versions) has the tag name with the channel suffix like `v0.0.1-alpha` and causes nuts to not do version comparisons properly for whatever reason (`Error: Version not found : ...`).

With this change, the only difference is that `tag` in each release object will only have the number of the tag, like `v0.0.1` but will the correct channel, like `alpha` and allow updates with channels to work correctly.

I think we should merge this change since it doesn't affect anything other than allowing update channels, and once we figure out a way to revert it later without breaking this PR, we proceed with it :)